lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -263,3 +246,59 @@ function dosomething_rogue_rb_exists_in_rogue($rbid)
 {
   return db_query("SELECT * FROM {dosomething_rogue_reportbacks} rogue_rbs WHERE rbid = :rbid", array(':rbid' => $rbid))->fetchAll();
 }
+
+/**
+ * Transform reportback into the appropriate scheme to send to Rogue.
+ *
+ * @param $northstar_id - User's Northstar user id.
+ * @param $user - Drupal user object.
+ * @param $values - Reportback values to send to Rogue.
+ * @param $run - Campaign run id.
+ * @return array
+ */
+function dosomething_rogue_transform_reportback($northstar_id, $user, $values, $run)
+{
```

Comment:
this bracket belongs on the row above in drupal, there's a few other times in this PR it's wrong